scripts/RetentionScript/RetentionScript.ps1

```diff
@@ -0,0 +1,194 @@
+<#
+ .SYNOPSIS
+    This powershell module deletes backups from the azure storage.
+
+ .PARAMETER UserName
+    UserName associated with file share.
+
+ .PARAMETER FileSharePath
+    The file share path of the storage configured for backup.
+
+ .PARAMETER StorageType
+    It could be one of the two storages supported:
+    1. AzureBlob
+    2. FileShare
+
+ .PARAMETER DateTimeBefore(Required)
+    It is the date time value for deleting the backups before that time. It should be provided in the format(yyyy-MM-dd HH.mm.ssZ)
+    Example: 
+    $DateTimeBefore = [DateTime]::Now.ToString("yyyy-MM-dd HH.mm.ssZ")
+
+ .PARAMETER ConnectionString
+    Connection string to the azure storage configured for backup.
+
+ .PARAMETER Pass
```

Comment:
Suggestion (No need to change) You could have created separate parameter set for File Share and Azure storage

Reply:
yes. I should have done that.